ow_lander/scripts/activity_grind.py

```diff
@@ -65,19 +65,6 @@ def grind(move_arm, move_limbs, move_grinder, args):
   h = math.sqrt( pow(y_start-constants.Y_SHOU,2) + pow(x_start-constants.X_SHOU,2) )
   l = constants.Y_SHOU - constants.HAND_Y_OFFSET
   beta = math.asin (l/h)
-
-  joint_goal = move_arm.get_current_joint_values()
-  joint_goal[constants.J_DIST_PITCH] = 0
-  joint_goal[constants.J_HAND_YAW] = 4*math.pi/3
-  joint_goal[constants.J_PROX_PITCH] = -math.pi/2
-  joint_goal[constants.J_SHOU_PITCH] = math.pi/2
-  joint_goal[constants.J_SHOU_YAW] = alpha + beta
-  # If out of joint range, abort
-  if (is_shou_yaw_goal_in_range(joint_goal)==False):
-    return False
-  move_arm.go(joint_goal, wait=True)
-  move_arm.stop()
-
```

Comment:
What these lines removed as part of this PR?

Reply:
In the test procedure you specify to test the _stow_ but this part touches the _grind_ command

Reply:
I saw that you put the reason for removing these lines in the PR description